clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/AbstractGoogleCachingAgent.groovy

```diff
@@ -37,13 +37,34 @@ abstract class AbstractGoogleCachingAgent implements CachingAgent, AccountAware
 
   GoogleCloudProvider googleCloudProvider
   String googleApplicationName // "Spinnaker/${version}" HTTP header string
-  String accountName
   GoogleNamedAccountCredentials credentials
-  // TODO(ttomsu): Make project and compute dynamic getters from credentials object.
-  String project
-  Compute compute
   ObjectMapper objectMapper
 
+  @VisibleForTesting
```

Comment:
you probably want an `@PackageScope` on here too if you really only want this accessible for a test case (its public by default in the wild wild groovy west)